homeassistant/core.py

```diff
@@ -1391,6 +1400,7 @@ def as_dict(self) -> Dict:
             "version": __version__,
             "config_source": self.config_source,
             "safe_mode": self.safe_mode,
+            "start_complete": self.start_complete,
```

Comment:
should this be a boolean or should we just include `hass.state` ?

Reply:
`hass.state` seems like it will stand up better with time assuming that works for @bramkragten

Reply:
I switched this out